openshift/e2e-common.sh

```diff
@@ -1,23 +1,29 @@
-#!/bin/sh 
```

Comment:
I do not fully agree on the name of this file.

What I'd like to see is that we have all the _setup related_ bits on our openshift/e2e-common, like:

* wait_until_machineset_scales_up
* installing serverless etc

But i wonder what is needed to be able to run the e2e test from the actual upstream file:
https://github.com/openshift/knative-eventing/blob/182bba7c990aa9e7b0a3fafc25164016c0acff2d/test/e2e-tests.sh

I think we would need overrides of `install_broker`, `uninstall_broker` and `install_mt_broker` 


But, if we do not need to "duplicate" the actual test instructions, just tweak the setup differenty, IMO this would be a win as well ? 

I thinking out loud here, but curious on feedback @mgencur @lberk

Reply:
@matzew not sure. The upstream stuff has functions such as `initialize` which we'd need to override to do something else. Trying to see the benefits of this approach but it might bring more pain than benefits. The only stuff we need from that upstream file are the lines starting with `go_test_e2e` ? Sounds like we'd need to override more stuff than we actually use from that file :-/

Reply:
What should be the name of the file? And why it's not good?